notebooks/500_use_cases/501_dobot/501b_Training_a_model_with_cubes_from_a_robotic_arm.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>If we need to show the output of the cells, I suggest to show just the config file, and the image of one cube and the images with results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/927/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>